ansible/roles/test/files/ptftests/dir_bcast_test.py

```diff
@@ -124,15 +121,57 @@ def check_ip_dir_bcast(self, dst_bcast_ip, dst_port_list):
         '''
         logging.info("Received " + str(pkt_count) + " broadcast packets, expecting " + str(len(dst_port_list)))
         assert (pkt_count == len(dst_port_list))
-    
+
+        return
+
+    #---------------------------------------------------------------------
+
+    def check_bootp_dir_bcast(self, dst_bcast_ip, dst_port_list):
```

Comment:
To me it looks like most of the functionality except the packet is same for both functions. We could create a single function for common code and use these functions only to create the packet. Approving from my side!